modules/microshift-install-multus-rpm.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-multus_{context}"]
+== Installing the multiple networks plugin
+
+Use this procedure to install the {microshift-short} Multus CNI plugin alongside a new {microshift-short} installation. The {microshift-short} Multus Container Network Interface (CNI) plugin is not installed by default. If you want to attach additional networks to a pod for high-performance network configurations, install the `microshift-multus` RPM package.
+
+[IMPORTANT]
+====
+The Multus RPM cannot be removed after you install it.
```

Comment:
There's no mechanism that will allow us to prevent uninstalling it so `cannot` or `should not`?